OpenTabletDriver.Configurations/Configurations/Gaomon/M7.json

```
@@ -0,0 +1,35 @@
+{
+  "Name": "Gaomon M7",
+  "Specifications": {
+    "Digitizer": {
+      "Width": 258.4,
+      "Height": 161.5,
```

Rounding to 200 LPMM based off `MaxX`/`MaxY` values:
```suggestion
      "Width": 258.445,
      "Height": 171.54,
```